README.md

```diff
@@ -33,8 +33,56 @@ We are actively planning and would love your feedback! Please create GitHub issu
 
 ----
 
-Pion WebRTC is a pure Go implementation of WebRTC. It has zero non-Go dependencies and no 3rd party Go dependencies. It is designed to follow **[WebRTC API](https://w3c.github.io/webrtc-pc/)**, but may deviate when required.
-See [DESIGN.md](DESIGN.md) for the guiding principals/inspirations of the project.
+### Features
+#### PeerConnection API
+* Go implementation of [webrtc-pc](https://w3c.github.io/webrtc-pc/) and [webrtc-stats](https://www.w3.org/TR/webrtc-stats/)
+* DataChannels
+* Send/Receive audio and video
+* Renegotiation
+* Plan-B and Unified Plan
+* [SettingEngine](https://pkg.go.dev/github.com/pion/webrtc/v3#SettingEngine) for Pion specific extensions
+
+#### Pure Go
```

Comment:
minor feedback: this section in between the webrtc specific functionality seems a bit out of place

Reply:
moved to the bottom!